### PR TITLE
[Feature] gelato stoploss order contract integration

### PR DESCRIPTION
**Pull request by HarryTgerman**
Integration UX: 
Bildschirmfoto 2021-11-11 um 11 16 11

### CODE REVIEW

File: packages/limit-orders-lib/src/constants/index.ts [hunk 1]
```diff
@@ -78,7 +85,7 @@ export const SUBGRAPH_URL = {
     "https://api.thegraph.com/subgraphs/name/gelatodigital/limit-orders-bsc",
 };
 
-export const GELATO_LIMIT_ORDERS_ADDRESS = {
+export const GELATO_CORE_ADDRESS = {
```
Comment (gitpusha) [inline packages/limit-orders-lib/src/constants/index.ts:88]: please dont change the name here

File: packages/limit-orders-lib/src/types/index.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ export type Handler =
   | "polydex"
   | "cafeswap"
   | "pancakeswap"
+  | "quickswap_stoploss";
```
Comment (pedrocrvz) [inline packages/limit-orders-lib/src/types/index.ts:15]: need to remove `;` here

**Comment by HarryTgerman** [comment]
@gitpusha @pedrocrvz, in `packages/limit-orders-lib/src/stoplimit-orders/index.ts > encodeStopLimitOrderSubmissionWithSecret ` i actually put in the `getFeeAndSlippageAdjustedMinReturn` value in the vault. Is that correct? What value should i display on the cards as MaxReturn? The vault value, or should i add the slippage + Gelato Fee back on? 
Bildschirmfoto 2021-11-19 um 19 30 59

**Comment by pedrocrvz** [comment]
> @gitpusha @pedrocrvz, in `packages/limit-orders-lib/src/stoplimit-orders/index.ts > encodeStopLimitOrderSubmissionWithSecret ` i actually put in the `getFeeAndSlippageAdjustedMinReturn` value in the vault. Is that correct? What value should i display on the cards as MaxReturn? The vault value, or should i add the slippage + Gelato Fee back on?
> 
> **Stoploss at 1 Matic for 1 USDC/ Slippage 2% :** Bildschirmfoto 2021-11-19 um 19 30 59

Not that familiar with the stop limit system, but on LOs, in the history card, we show the `minReturn` inputed by the user - i.e the value submitted on chain + slippage + fees (LO example below, where you can the submission + the history card).

Screenshot 2021-11-22 at 14 58 49

So in your example, assuming the same, it should be `Sell 1 MATIC for 1 USDC`. (also, there's a weird value on your max execution price)

**Comment by HarryTgerman** [comment]
Closed because this feature will be released in [swap-orders-lib](https://github.com/gelatodigital/swap-orders-lib) repo

**Pull request closed without merging**